src/tileset.rs

```diff
@@ -134,6 +137,37 @@ impl Tileset {
         self.external_file.as_ref()
     }
 
+    pub(crate) fn image(&self, image_pixels: Vec<image::Rgba<u8>>) -> RgbaImage {
```

Comment:
Small nit. Since this is an internal function with the same name as a global API function, it would be nice if this would have a more internal-sounding name. E.g., more along the lines of `tileset_image_from_pixels` (not great). Or even `tilemap_image`. Maybe you can think of something better.

Reply:
I decided on `write_to_image` to match similar internal functions that return `RgbaImage`.